recipes/fgbio/meta.yaml

```diff
@@ -11,11 +11,11 @@ build:
   number: 0
   skip: false
 source:
-  fn: fgbio-{{ tag }}.tar.gz
-  url: https://github.com/fulcrumgenomics/fgbio/archive/{{ tag }}.tar.gz
-  #fn: fgbio-{{ version }}.jar
-  #url: https://github.com/fulcrumgenomics/fgbio/releases/download/{{ version }}/fgbio-{{ version }}.jar
-  sha256: 7c272dbc89bbac5782a2f04234fa6e6ede9883558b749e6af0bbeeef4e8dc488
+  # fn: fgbio-{{ tag }}.tar.gz
+  # url: https://github.com/fulcrumgenomics/fgbio/archive/{{ tag }}.tar.gz
+  fn: fgbio-{{ version }}.jar
+  url: https://github.com/fulcrumgenomics/fgbio/releases/download/{{ version }}/fgbio-{{ version }}.jar
+  sha256: c35286ccf82158ffcc8944a063461b9aa8636499bb99f9df92a592adba95c854
```

Comment:
are the build dependencies still needed?

Reply:
Thank you